src/lib/ApxData/Sharing/Share.cpp

```diff
@@ -94,6 +94,12 @@ void Share::exportTriggered()
 
     QFileInfo fi(dlg.selectedFiles().first());
 
+    QSettings st;
+    st.beginGroup("plots");
```

Comment:
should not be used here, as `Share` is a base class for other sharing tools and not related to `plots`.